src/ui/services/networkService.ts

```diff
@@ -23,50 +23,72 @@ export class NetworkService extends EventEmitter {
     /** Current Socket.IO connection to central server. */
     currentConnection: SocketIOClient.Socket;
 
+    /** List of all users. */
+    users: user.User[];
+
+    me: user.User;
+
+    imHost: boolean;
```

Comment:
`isHost`

Reply:
Also, this doesn't seem to be used anywhere.

Reply:
d